files/playbooks/2023.1/kolla-common.yml

```diff
@@ -1,6 +1,6 @@
 ---
 - name: Apply role common
-  hosts: all
+  hosts: "{{ hosts_default_group|default('generic') }}"
```

Comment:
Use hosts_kolla_default_group as parameter name and use all as default to not change the default value.

Reply:
Changed that for all plays.